adobe/spectrum-web-components
# fix: add likeAnchor API to Card element

## Description
Apply the `likeAnchor` mixin to the `Card` class so that a card can be linkable.

### To-do:
- [ ] add additional stories?
- [ ] update golden image cache

## Related issue(s)

- fixes #1743 
- refs #1281 

## Motivation and context
Applying this from the outside is fraught with chances to mess up the accessibility of the Card element and its inclusion makes consuming the library much easier.

## How has this been tested?

    1. Go to https://card-like-anchor--spectrum-web-components.netlify.app/storybook/index.html?path=/story/card--href&args=size:!undefined;horizontal:false
    2. Click the card
    3. See the `link click` report in the "Actions" tab.
    1. Go to https://card-like-anchor--spectrum-web-components.netlify.app/storybook/index.html?path=/story/card--href&args=size:!undefined;horizontal:false
    2. Click the "link to Google"
    3. See that you make it to google.com
    1. Go to https://card-like-anchor--spectrum-web-components.netlify.app/storybook/index.html?path=/story/card--href&args=size:!undefined;horizontal:false
    2. Click the Action Menu
    3. Select an Item
    4. Ensure that none of the interactions report a `link click` in the Actions tab

## Types of changes

## Checklist

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
